client/src/components/Settings/SettingsProgressPath/SettingsProgressPath.tsx

```diff
@@ -0,0 +1,64 @@
+import cx from 'classnames';
+import { motion } from 'framer-motion';
+import React, { FC, useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import styles from './SettingsProgressPath.module.scss';
+
+const SettingsProgressPath: FC = () => {
```

Comment:
No children, no props, no need for an `FC`.
```suggestion
const SettingsProgressPath: (): ReactElement => {
```

Reply:
It has props in next PRs